.storybook/docgen-plugin.js

```diff
@@ -106,15 +108,15 @@ const processModule = (module, tsProgram) => {
   let source = module._source._value;
   source +=
     '\n' +
-    docGenLoader
-      .default({
-        filename: module.userRequest,
-        source: module.userRequest,
-        componentDocs,
-        docgenCollectionName: 'STORYBOOK_REACT_CLASSES',
-        setDisplayName: true,
-      })
-      .substring(module.userRequest.length) +
+    generateDocgenCodeBlock({
+      filename: module.userRequest,
+      source: module.userRequest,
+      componentDocs,
+      docgenCollectionName: 'STORYBOOK_REACT_CLASSES',
+      setDisplayName: true,
+      savePropValueAsString: true,
+      typePropName: 'type',
```

Comment:
This was the value that was added as an optional param with a default. Since we don't use the default export and use a private one instead, this default isn't being set (even if it was released as a minor...). This was needed even with `react-docgen-typescript-loader`. I decided to keep `@storybook/react-docgen-typescript-plugin` since it is being actively updated and probably handles future Storybookisms. I may be around long enough to regret this decision, however.